<ui version="4.0" >
 <class>QgsWKNDiagramFactoryWidgetBase</class>
 <widget class="QWidget" name="QgsWKNDiagramFactoryWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="mAttributesLabel" >
       <property name="text" >
        <string>Attributes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mAttributesComboBox" />
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" colspan="2" >
      <widget class="QTreeWidget" name="mAttributesTreeWidget" >
       <property name="columnCount" >
        <number>2</number>
       </property>
       <column>
        <property name="text" >
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="mRemovePushButton" >
       <property name="text" >
        <string>Remove attribute</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QPushButton" name="mAddPushButton" >
       <property name="text" >
        <string>Add attribute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
